sbi/analysis/sensitivity_analysis.py

```diff
@@ -172,58 +172,56 @@ def add_property(
         self._theta = theta
         self._emergent_property = emergent_property
 
+        def build_resnet(theta):
```

Comment:
move to outside this function?

Reply:
they rely on variables that are available only within the scope of the function. So it would be a bit of a hassle